practice-two/src/js/controllers/product.controller.js

```diff
@@ -0,0 +1,18 @@
+import HttpService from "../services/httpClient.service";
```

Comment:
Because ProductController is already in controllers/ folder so I think just keep the file as product.js for simpler

Reply:
same with views and models